tests/vm_test.go

```diff
@@ -2122,7 +2123,13 @@ func stopVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.Virtua
 }
 
 func retryWithMetadataIfModified(objectMeta metav1.ObjectMeta, do func(objectMeta metav1.ObjectMeta) error) (err error) {
-	return tests.RetryIfModified(func() error {
-		return do(objectMeta)
-	})
+	retries := 0
+	for err = do(objectMeta); errors.IsConflict(err); err = do(objectMeta) {
```

Comment:
nit: can we split this line? It is complicated to read and it is doing quite a lot.

Reply:
This was just a copy-paste from `utils.go`. I don't think refactor is needed because I'll remove this `retryWithMetadataIfModified` function in a new PR after it, currently this function is used by 2 functional tests that one of them irrelevant and another one can be modified.

Reply:
ok sounds good